pkgs/shells/antigen/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "2.2.1";
+  name = "antigen";
+
+  src = fetchurl {
+    url = "git.io/antigen";
```

Comment:
There are releases, I can fetch file from there. Or clone a repo and copy all needed stuff from there